src/ace/managers/log.c

```diff
@@ -165,17 +165,17 @@ void _logAvgEnd(tAvg *pAvg) {
 		pAvg->ulMin = pAvg->pDeltas[pAvg->uwCurrDelta];
 	++pAvg->uwCurrDelta;
 	// Roll
-	if(pAvg->uwCurrDelta == pAvg->uwAllocCount)
+	if(pAvg->uwCurrDelta >= pAvg->uwAllocCount) {
```

Comment:
In every project I'm involved I'm always prefer to use `<` and `<=` operators. Making this consistent enables faster reading as increasing X value on typical chart from math classes.

Reply:
I prefer typing it in a way that's naturally coming to your mind. You'd rather say that "var has reached or passed its limit", than a "limit is equal or smaller than var".